docker/service/start_service.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# start_service.sh
+# Start a full Codalab Worksheets Server service
+
+set -a
+
+usage()
+{
+  echo "Starts a full Codalab Worksheets Server service. Optionally builds docker images for it. If not building local images, 'latest' tags used, otherwise images are built with the 'local-dev' tag and these images are used. [[-b --build: Build docker images first] [-h --help: get usage help]]"
+}
+
+BUILD=0
+INIT=0
+
+CODALAB_MYSQL_ROOT_PWD=${CODALAB_MYSQL_ROOT_PWD:-mysql_root_pwd}
+
+CODALAB_MYSQL_USER=${CODALAB_MYSQL_USER:-bundles_user}
+CODALAB_MYSQL_PWD=${CODALAB_MYSQL_PWD:-mysql_pwd}
+
+CODALAB_ROOT_USER=${CODALAB_ROOT_USER:-codalab}
+CODALAB_ROOT_PWD=${CODALAB_ROOT_PWD:-testpassword}
+
+CODALAB_SERVICE_HOME=${CODALAB_SERVICE_HOME:-/var/lib/codalab/home/}
+CODALAB_BUNDLE_STORE=${CODALAB_BUNDLE_STORE:-/var/lib/codalab/bundles/}
+CODALAB_MYSQL_MOUNT=${CODALAB_MYSQL_MOUNT:-/var/lib/codalab/mysql/}
+
+
+for arg in "$@"; do
+  case $arg in
+    -b | --build )      BUILD=1
+                        ;;
+    -i | --init )       INIT=1
+                        ;;
+    -h | --help )       usage
+                        exit
+  esac
+done
+
+if [ "$BUILD" = "1" ]; then
+  CODALAB_VERSION=local-dev
+  echo "==> Building 'local-dev' Docker images"
+  ./docker/build_images.sh local-dev
```

Comment:
For all of these commands, do we want to put a `|| exit 1` after each one to not continue if something fails?